hokusai/development.yml

```diff
@@ -5,14 +5,21 @@ services:
 {% include 'templates/docker-compose-service.yml.j2' %}
     environment:
       - NODE_ENV=development
-      - REDIS_URL=redis://force-redis:6379/0
+      - OPENREDIS_URL=redis://force-redis:6379/0
```

Comment:
Weird... I actually don't see any references to the removed `REDIS_URL` in the codebase, but both staging and production provide it. Is there a chance libraries depend on it directly?

Reply:
not sure. added it back just to be safe.